savant/client/runner/source.py

```diff
@@ -170,41 +166,69 @@ def send_iter(
             for source_id in source_ids:
                 self.send_eos(source_id)
 
-    def send_eos(self, source_id: str) -> SourceResult:
+    def send_eos(self, zmq_topic: str) -> SourceResult:
```

Comment:
The argument should be named `source_id`, not `zmq_topic` since its sending end of stream, which is identified by `source_id`.